### ISSUE TITLE
vite-plugin-ruby@^5.0.0 might not be compatible with vite_ruby-3.8.2

### ISSUE THREAD

**Issue by Sergyenko** [maglevhq/maglev-core#86]
Hello, I'm trying to follow the quickstart doc:

https://docs.maglev.dev/quickstart

But getting the following errors:
```
rails generate maglev:install
/Users/sergy/.asdf/installs/ruby/3.3.3/lib/ruby/gems/3.3.0/gems/vite_ruby-3.8.2/lib/vite_ruby/compatibility_check.rb:24:in `raise_unless_satisfied': vite-plugin-ruby@^5.0.0 might not be compatible with vite_ruby-3.8.2 (ArgumentError)

You may disable this check if needed: https://vite-ruby.netlify.app/config/#skipcompatibilitycheck

You may upgrade both by running:

    bundle exec vite upgrade

        raise ArgumentError, <<~ERROR
              ^^^^^^^^^^^^^^^^^^^^^^^
```
The upgrade command didn't help, as well as disable key to config/vite.json

I'm using Ruby 3.3.3, and Rails 7.2.1 on Mac M3

Happy to help to fix it, but would appreciate any hint :)

**Comment by did** [comment]
thanks @Sergyenko! I faced the same issue a couple of days ago. The `vite_ruby` gem was recently updated. I'm going to fix that.

**Comment by Sergyenko** [comment]
I've used the following key:

`export VITE_RUBY_SKIP_COMPATIBILITY_CHECK=true`

And it worked well. Also tried to use the latest gem version, which was good as well `gem 'vite_ruby', '4.0.0.alpha1'`

Let me know how it goes and/or any help needed - I'm happy to contribute 🙏

**Comment by did** [comment]
Alright, I've released a new version (1.7.3) with my fix. Thanks again @Sergyenko.

**Comment by Sergyenko** [comment]
Thank you! Now all works just great 🙏
